raven-js/index.d.ts

```diff
@@ -1,212 +1,305 @@
 // Type definitions for Raven.js
 // Project: https://github.com/getsentry/raven-js
-// Definitions by: Santi Albo <https://github.com/santialbo/>, Benjamin Pannell <http://github.com/spartan563>, Gary Blackwood <http://github.com/Garee>, Rich Rout <http://github.com/richrout>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Original Definitions by: Santi Albo <https://github.com/santialbo>, Benjamin Pannell 
+//   <http://github.com/spartan563>; DefinitelyTyped
+// Updated by: Ben Vinegar <https://github.com/benvinegar>, Ilya Pirogov 
```

Comment:
Fair enough, didn't know about that. The error message is complaining about the missing `Definitions by:`.